gcp/migrations/verify/appschema.sql

```diff
@@ -0,0 +1,7 @@
+-- Verify golf-better:appschema on pg
+
+begin;
+
+select pg_catalog.has_schema_privilege('golfbetter', 'usage');
```

Comment:
sqitch has these "verify" scripts in addition to the usual deploy+revert scripts. We can setup another workflow to verify our deployed DB as well to ensure the schema is in alignment (and also auto-deploy schema changes to dev), but that will be another PR